grpc-api/src/main/java/io/deephaven/grpc_api/table/ops/filter/FilterFactory.java

```diff
@@ -173,7 +173,7 @@ public SelectFilter onIn(Value target, List<Value> candidatesList, CaseSensitivi
             if (literal.getValueCase() == Literal.ValueCase.NANO_TIME_VALUE) {
                 values[i] = "'" + new DBDateTime(literal.getNanoTimeValue()).toString(DBTimeZone.TZ_DEFAULT) + "'";
             } else {
-                values[i] = FilterPrinter.printNoEscape(literal);
+                values[i] = FilterPrinter.print(literal);
```

Comment:
This will break with characters that must be escaped, the noescape variant is expected to wrap in double quotes, but still leave the output unescaped. For better or worse, MatchFilter expects double quote wrapping the strings, but ignores any other escaping inside the string, we should consider changing this behavior.